src/client/game.cpp

```diff
@@ -2001,7 +2002,7 @@ void Game::processItemSelection(u16 *new_playeritem)
 
 	/* Item selection using hotbar slot keys
 	 */
-	for (u16 i = 0; i < 23; i++) {
+	for (u16 i = 0; i < HUD_HOTBAR_ITEMCOUNT_MAX; i++) {
```

Comment:
```suggestion
	for (u16 i = 0; i <= max_item; i++) {
```
So you can get rid of the unnecessary range check two lines below.

Reply:
Sounds okay I guess, but can't `max_item` sometimes have a different value than `HUD_HOTBAR_ITEMCOUNT_MAX` ?

C++ definitely isn't my forte. :smiley: